### PR TITLE
Fix PropertyValueController's FormRequest Errors #77 

### PR DESCRIPTION
**Pull request by S-DICKSON**
PR for PropertyValueController's FormRequest Errors #77.

The styleci is failing not sure how to fix it.

Also not run tests so any guidance on how to do this would be amazing thank you.

Update::
I've created this change today as I currently have skipped a test I want to solve.

I've improved the general security of the PropertyValueController as they were creating models using $request->all().

https://www.youtube.com/watch?v=QQS5oEOguRU

https://laravel.com/docs/8.x/validation#creating-form-requests

```
    // Retrieve the validated input data...
    $validated = $request->validated();
```
Changed UpdatePropertyValue.php  to require property_id so no SQL error is visible to the user.

### CODE REVIEW

**Comment by fulopattila122** [comment]
Have you read this?

> Please go through this checklist, it is mandatory for your PR to be accepted:
> 
> Remove this default text
> Include the summary of your change.
> Make sure all the tests are passing
> Make sure StyleCI is passing
> In case you add new functionality, it must be covered by tests

**"Please go through this checklist, it is mandatory for your PR to be accepted"**

In other words, no review will happen until it gets done.
In particular, please let us know what is the reason behind the change.

**Comment by fulopattila122** [comment]
> The styleci is failing not sure how to fix it.
Click the red X next to the commit and it will tell you the details.
Also, please remove any unnecessary comments or surplus lines.

> Also not run tests so any guidance on how to do this would be amazing thank you.

```bash
vendor/bin/phpunit
```

File: src/Http/Controllers/PropertyValueController.php [hunk 1]
```diff
@@ -35,12 +35,15 @@ public function create(CreatePropertyValueForm $request, Property $property)
         ]);
     }
 
+    /**
+     * @refactored 07-07-2021 sd@groundwow.com
+     */
```
Comment (fulopattila122) [inline src/Http/Controllers/PropertyValueController.php:40]: Please remove @refactored comments from the PR

Comment (S-DICKSON) [reply src/Http/Controllers/PropertyValueController.php:40]: Will do

**Comment by S-DICKSON** [comment]
> Have you read this?
> 
> > Please go through this checklist, it is mandatory for your PR to be accepted:
> > Remove this default text
> > Include the summary of your change.
> > Make sure all the tests are passing
> > Make sure StyleCI is passing
> > In case you add new functionality, it must be covered by tests
> 
> **"Please go through this checklist, it is mandatory for your PR to be accepted"**
> 
> In other words, no review will happen until it gets done.
> In particular, please let us know what is the reason behind the change.

Hi yeah, I did read it.  My reason for the change is the PR request and issue number on the description. I will expand on this now.

**Comment by S-DICKSON** [comment]
I did this.
```
git clone https://github.com/S-DICKSON/framework.git

cd framework

composer install

sudo apt-get install php7.4-sqlite

vendor/bin/phpunit
```

image

**Pull request merged**